site/logsdb-engine/js/app.js

```
@@ -0,0 +1,508 @@
+      setText("stat-total-bytes", formatBytes(storeStats.totalChunkBytes));
+      setText("stat-ingest-rate", `${formatNum(ingestResult.logsPerSecond)} logs/s`);
+      setText("stat-gen-time", `${totalTime.toFixed(0)}ms`);
+
+      show("stats-panel");
+      show("tabs-panel");
+      hide("gen-progress");
+
+      // Render initial tab
+      renderCurrentTab();
+
+      buttons.forEach((b) => (b.disabled = false));
```

Biome (2.4.13)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)